third-party/CMakeLists.txt

```diff
@@ -106,7 +134,11 @@ if(USE_BUNDLED_LUAROCKS)
 endif()
 
 if(USE_BUNDLED_JEMALLOC)
-  include(BuildJeMalloc)
+  if (WIN32)
+    message(STATUS "Building jemalloc in Windows is not supported (skipping)")
+  else()
+    include(BuildJeMalloc)
```

Comment:
No idea - related discussion https://bugzilla.mozilla.org/show_bug.cgi?id=427107 

I think I can build standard jemalloc with Mingw but not for the current release. Whether or not that is safe to use is an different issue :)